routes/api/v1/twilio.js

```diff
@@ -35,6 +37,14 @@ const smsText = [
   }
 ]
 
+function formatPhoneNumber(input) {
```

Comment:
seems like a useful function that may come in handy in multiple files, maybe move to like a `helpers/utils.js` file later or whatever makes sense